iOS/docs/kickoff/README.md

```diff
@@ -1,4 +1,157 @@
+
+A little heads up: It's *extremely* important naming conventions, such as camel case notation and so on, are followed. I.e.: if `BaseProject.xcodeproj` should be renamed with our project name, then it should be `ProjectName.xcodeproj`. Always follow the naming pattern on the base project. Also, this is meant to be sequentially, so don't skip or mix steps.
+
+### Setup
+
+#### Adding Base Project
+
+First, clone the base project repository. The included project comes with a basic XCode configuration and includes the most useful/needed dependencies. To do this, run on your Terminal:
+
+`$ git clone git@github.com:Wolox/ios-base-project.git`
+
+We should then clone from GitHub the empty project repository for our `Project Name`, with ULR `https://github.com/Wolox/example-project-ios` and SSH URL `git@github.com:Wolox/project-name-ios.git`. On the same path/console, run:
+
+`$ git clone git@github.com:Wolox/project-name-ios.git`
+
+Clone the repository.
+
+`$ rsync -av --progress ios-base-project/. project-name-ios --exclude .git`
+
+Fast copy everything but the .git file from base project to your project.
+
+`$ cd project-name-ios`
+
+Change to your project directory.
+
+`$ git add .`
+
+Don't forget to add the changes.
+
+`$ git commit -m "Initial commit"`
+
+Watch the message! This will be your project's first commit ;)
+
+`$ git push origin HEAD -u`
+
+Then send it to the clouds!
+
+When these is done a new branch should be created to properly set up the project. Once again, run on your terminal:
+`$ git checkout -b "project-setup" `
+
+#### Project Renaming
+
+Open XCode project by running:
+
+`$ open BaseProject.xcodeproj`
+
+Change its name from `BaseProject` to your own `ProjectName`, following [this]((https://developer.apple.com/library/ios/recipes/xcode_help-project_editor/RenamingaProject/RenamingaProject.html)) tutorial.
+
+After doing this you'll note that still the `ProjectName`'s `Scheme` hasn't been renamed. To do this, duplicate the actual scheme and rename it, only then delete the old scheme. While editing the scheme, tick the `shared` option so project can be run on CI environment.
+
+![rename scheme](./resources/xcode-scheme-rename.gif)
+
+Folders and groups should also be renamed. Run the following on your console to fasten the process:
+
+`$ brew install rename ack`
+`$ find . -name 'BaseProject*' -print0 | xargs -0 rename -S 'BaseProject' 'ProjectName'`
+`$ ack --literal --files-with-matches 'BaseProject' | xargs sed -i '' 's/BaseProject/ProjectName/g'`
+
+Verify every `BaseProject` appearance has been renamed, running:
+
+`$ ack --literal 'BaseProject'`
+
+If this hasn't print out anything, you are good to go!
+
+#### Build Scripts Configuration
+
+Renaming is over! Time to move on to configuring the [build scripts](https://github.com/guidomb/ios-scripts) file. To do this, edit the `script/.env` file, which should already contain the following:
+
+```
+REQUIRED_SWIFTLINT_VERSION=0.9.2
```

Comment:
Sure, I just deleted the content of the file, since it's already in `ios-base-project` and so we don't have to update there and there over and over.